home-data-migration/src/main/resources/db/changelog/0.0.1/Issue304.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <changeSet id="Issue304.001" author="klaeck">
+        <dropColumn tableName="invitations" columnName="cooperation_name"/>
+        <addColumn tableName="invitations">
+            <column name="cooperation_id" type="bigint">
+                <constraints nullable="true" foreignKeyName="cooperation_fk" references="cooperations(id)"/>
```

Comment:
Can we 

1. Add column cooperation_id w/o constraints
2. Fill cooperation_id based on cooperation_name
3. Add constraints to cooperation_id, make it nullable false
4. Drop cooperation_name